src/environments/environment.ts

```diff
@@ -5,7 +5,7 @@
 
 export const environment = {
   production: false,
-  // API_ENDPOINT: 'http://localhost:3000/',
-  API_ENDPOINT: 'https://angularspree-new.herokuapp.com/',
+  API_ENDPOINT: 'http://localhost:3000/',
```

Comment:
@laertispappas This should not be committed, small thing 👍

Reply:
Done. Thanks